corehq/form_processor/generic.py

```diff
@@ -201,6 +204,10 @@ def reverse_indices(self):
         from corehq.form_processor.interfaces.case import CaseInterface
         return CaseInterface.get_reverse_indices(self.domain, self.id)
 
+    @property
+    def is_deleted(self):
+        return self.doc_type.endswith(DELETED_SUFFIX)
```

Comment:
agreed, fiddled with doing that for a moment and then just went the easy route. marked it down to change though